api/config.py

```diff
@@ -104,3 +104,15 @@
     'image_url': f"{WWW_BASE_URL}/badges/skin-in-the-game.png",
     'price_usd': 0.21 if ENV == 'staging' else 21,
 }
+
+LNDHUB_URL = os.environ.get('LNDHUB_URL', "https://ln.getalby.com")
+LNDHUB_USER = os.environ.get('LNDHUB_USER')
+LNDHUB_PASSWORD = os.environ.get('LNDHUB_PASSWORD')
+
+if LNDHUB_USER is None or LNDHUB_PASSWORD is None:
```

Comment:
I don't think we should load secrets if these values are None. because there are some valid cases where we do want them to be None, such as testing... 

so the decision to load secrets or not should be based on a `MOCK_LNDHUB` boolean...

Reply:
But it would still be useful to be able to specify the values in the secrets directory (for deploy to environments), and having the values in `.env.dev` for use in localhost, right?